config/docker-compose-worker.properties

```diff
@@ -8,15 +8,9 @@ bootstrap.servers=kafka:9092
 # when loaded from or stored into Kafka
 key.converter=org.apache.kafka.connect.converters.ByteArrayConverter
 key.converter.schema.registry.url=http://confluent:8081
-value.converter=org.apache.kafka.connect.converters.ByteArrayConverter
-value.converter.schema.registry.url=http://confluent:8081
-# The internal converter used for offsets and config data is configurable and must be specified,
-# but most users will always want to use the built-in default. Offset and config data is never
-# visible outside of Connect in this format.
-internal.key.converter=org.apache.kafka.connect.json.JsonConverter
-internal.value.converter=org.apache.kafka.connect.json.JsonConverter
-internal.key.converter.schemas.enable=false
-internal.value.converter.schemas.enable=false
+value.converter=io.confluent.connect.avro.AvroConverter
```

Comment:
I think `ByteArrayConverter` should remain the default

Reply:
Good point - Reverted it 7011ca1b67c0537814c4b141f39228514ac310e3